src/ai/backend/client/cli/session/lifecycle.py

```diff
@@ -844,48 +846,63 @@ def logs(session_id: str, kernel: str | None) -> None:
             sys.exit(ExitCode.FAILURE)
 
 
-@session.command("status-history")
+@session.command()
```

Comment:
I don't know much about `session.command`, but can you subtract the internal string "status-history"?

Reply:
https://github.com/lablup/backend.ai/blob/42b4030e9aecd32e297e8b5b08fad4729a2ea2a9/src/ai/backend/client/cli/session/lifecycle.py#L59-L61

Looking at the code, it seems that the decorator itself does not perform any operations on the arguments.
So, I think it would be better to leave the arguments empty, as is currently done in most of the other code.

Reply:
I think this is using `overload` to overload the function for the last implementation, can you double check? @jopemachine

Reply:
If no value is passed as an argument, the decorated function name is automatically used as the CLI command.
If we pass an arbitrary string, that string can be used as the CLI command instead of changing the function name.
So, let's leave the arguments empty here, just like with the other functions.